movies/movies

```diff
@@ -170,7 +170,7 @@ getConfiguredPython || exit 1
 getConfiguredClient || exit 1
 checkInternet || exit 1 # check if we have a valid internet connection if this isnt true the rest of the script will not work so stop here
 
-while getopts 'ud:hv' flag; do
+while getopts 'udhv' flag; do
```

Comment:
```ud:hv``` in conjunction with
```bash
 :)
     echo "Option -$OPTARG requires an argument." >&2
     exit 1
   ;;
```
will create the error message that you are looking for. Without one of those pieces it shouldn't work.